src/elements/content-uploader/ContentUploader.js

```diff
@@ -71,6 +71,7 @@ type Props = {
     onComplete: Function,
     onError: Function,
     onMinimize?: Function,
+    onResume: Function,
```

Comment:
This should probably be added to the documentation in the readme

Reply:
👍